pycbc/io/hdf.py

```diff
@@ -998,6 +997,56 @@ def to_coinc_xml_object(self, file_name):
 
         ligolw_utils.write_filename(outdoc, file_name)
 
+    def to_coinc_hdf_object(self, file_name):
+        ofd = h5py.File(file_name,'w')
+
+        # Some fields are special cases:
+
+        time = self.get_end_time()
+        ofd.create_dataset('end_time', data=time, dtype=np.float32)
+
+        far = 1. / self.get_coincfile_array('ifar')
+        ofd.create_dataset('false_alarm_rate', data=far, dtype=np.float32)
+
+        far_exc = 1. / self.get_coincfile_array('ifar_exc')
+        ofd.create_dataset('false_alarm_rate_exclusive', data=far_exc,
+                           dtype=np.float32)
+
+        fap = self.get_coincfile_array('fap')
+        ofd.create_dataset('p_value', data=fap,
+                           dtype=np.float32)
+
+        fap_exc = self.get_coincfile_array('fap_exc')
+        ofd.create_dataset('p_value_exclusive', data=fap_exc,
+                           dtype=np.float32)
+
+        # Coinc fields
+        for field in ['stat']:
+            vals = self.get_coincfile_array(field)
+            ofd.create_dataset(field, data=vals, dtype=np.float32)
+
+        # Bank fields
+        for field in ['mass1','mass2','spin1z','spin2z']:
+            vals = self.get_bankfile_array(field)
+            ofd.create_dataset(field, data=vals, dtype=np.float32)
+
+        mass1 = self.get_bankfile_array('mass1')
+        mass2 = self.get_bankfile_array('mass2')
+        mchirp, _ = mass1_mass2_to_mchirp_eta(mass1, mass2)
+
+        ofd.create_dataset('chirp_mass', data=mchirp, dtype=np.float32)
+
+        # Single-detector fields
+        for field in ['chisq', 'chisq_dof', 'coa_phase', 'end_time',
+                      'sg_chisq', 'sigmasq', 'snr']:
+            vals_valid = self.get_snglfile_array_dict(field)
+            for ifo in self.ifos:
+                vals = vals_valid[ifo][0]
+                valid = vals_valid[ifo][1]
+                vals[np.logical_not(valid)] = -1.
+                ofd.create_dataset(ifo + '_'+field, data=vals,
+                                   dtype=np.float32)
```

Comment:
Can you add another special column here. `network_snr`, which would be the quadrature sum of the various `snr` columns?